libs/sdk-ui-dashboard/src/presentation/dashboard/types.ts

```diff
@@ -460,6 +461,14 @@ export interface IDashboardBaseProps {
      * logged-in user.
      */
     permissions?: IWorkspacePermissions;
+
+    /**
+     * Specify which render mode will be used for initial rendering.
+     *
+     * @remarks
+     * If you do not specify initialRenderMode, the dashboard component will be display in view mode.
+     */
+    initialRenderMode?: RenderMode;
```

Comment:
We should probably mark most of these new edit mode props and interfaces as `@internal` before edit mode GA, so if there will be some SDK relase meanwhile, it won't be public yet. @no23reason @lupko what do you think?